packages/Webkul/Shop/src/Resources/views/emails/layouts/master.blade.php

```diff
@@ -4,6 +4,13 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
         <link href="https://fonts.googleapis.com/css?family=Montserrat:400,500" rel="stylesheet" type="text/css">
+        <style>
+            .payment-method {
```

Comment:
hi @devansh-webkul 

Why are writing CSS in the master file and check failed test cases?